### PR TITLE
feat(RHINENG-7052): Update to FEC 

### PR DESCRIPTION
**Pull request by mkholjuraev**
# Description

Resolves: https://issues.redhat.com/browse/RHINENG-7052

# How to test the PR

1. Run the application
2. Make sure there are no changes and the app functions as expected

# Before the change

# After the change

# Dependent work link

# Checklist:

- [ ] The commit message has the Jira ticket linked
- [ ] PR has a short description
- [ ] Screenshots before and after the change are added
- [ ] Tests for the changes have been added
- [ ] README.md is updated if necessary
- [ ] Needs additional dependent work

### CODE REVIEW

File: fec.config.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    ? { hotReload: process.env.HOT === 'true' }
+    : { hotReload: true }),
+  ...(process.env.port ? { port: parseInt(process.env.port) } : {}),
+  moduleFederation: {
+    shared: [
+      {
+        'react-router-dom': {
+          singleton: true,
+          import: false,
+          version: '^6.8.1',
```
Comment (adonispuente) [inline fec.config.js]: This is more of a nitpick, but we can make this more dynamic with 

`const packageJson = require('./package.json');
` at the top and then replacing the version line with
`version: packageJson.dependencies['react-router-dom'],`

File: package.json [hunk 2]
```diff
@@ -3,26 +3,23 @@
   "version": "1.1.0",
   "private": false,
   "scripts": {
-    "build": "webpack --config config/prod.webpack.config.js",
-    "analyze": "NODE_ENV=production webpack --config config/prod.webpack.config.js --env analyze=true",
-    "build:prod": "NODE_ENV=production webpack --config config/prod.webpack.config.js",
+    "build": "fec build",
+    "build:prod": "NODE_ENV=production fec build",
```
Comment (adonispuente) [inline package.json]: We can keep this but not sure we need a build prod. I havent seen this in other apps.

**Review by adonispuente — APPROVED** [review]
overall LGTM, small nitpicks but this can be merged. App functionality is working as expected.

File: fec.config.js (hunk 1 above)
Comment (mkholjuraev) [reply fec.config.js]: Nice suggestion!

**Pull request merged**